### ISSUE TITLE
Generalize section settings inheritance

### ISSUE THREAD

**Issue by zenderol** [WeltN24/WeltContentApiClient#17]
There must be an easy way to add a field for a section (via ConfigMcConfigFace and Janus) which is inherited to all sub-sections. (e.g. /partner/, /partner/XYZ ...)

At the moment we have such a logic for defining an ad tag. To prevent writing similar code each time a new inherited field is added, we should implement a generic way for doing this.

**Comment by thisismana** [comment]
Can you be more specific about this? A use-case would be helpful. I fear that it is not always intended to automatically set a certain value for the current node **and** all child nodes, everytime you make a change.

**Comment by aramin** [comment]
Geht eigentlich nur darum, dass wir nicht ständig solche rekursiven Methoden neu implementieren müssen, sobald ein weiteres Feld hinzukommt, welches vererbt werden soll:
```
def getAdTag: Option[String] = {
    if (data.adData.definesAdTag) {
      val adPath: String = id.path.stripPrefix("/").stripSuffix("/").trim match {
        case empty if empty.isEmpty && id.ece == 5 => "home"
        case ok if !ok.isEmpty => ok
        case _ => DEFAULT_AD_TAG
      }
      Some(adPath)
    } else {
      parent.flatMap(_.getAdTag)
    }
  }
```

**Comment by bobaaaaa** [comment]
Fixed with #56 -- see docs
